instrumentation/jaxws/jaxws-3.0-common-testing/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+plugins {
+  id("org.unbroken-dome.xjc")
+  id("otel.java-conventions")
+}
+
+tasks {
+  named<Checkstyle>("checkstyleMain") {
+    // exclude generated web service classes
+    exclude("**/hello_web_service/**")
+  }
+}
+
+dependencies {
+  api("jakarta.xml.ws:jakarta.xml.ws-api:3.0.0")
+  api("jakarta.jws:jakarta.jws-api:3.0.0")
+
+  api("ch.qos.logback:logback-classic")
```

Comment:
`testing-common` should already add logback and slf4j dependencies, check whether you really need them here